pkgs/applications/networking/cluster/openshift/default.nix

```diff
@@ -15,10 +15,10 @@ stdenv.mkDerivation rec {
     owner = "openshift";
     repo = "origin";
     rev = "v${version}";
-    sha256 = "0zw8zb9c6icigcq6y47ppnjnqyghk2kril07bapbddvgnvbbfp6m";
+    sha256 = "0qvyxcyca3888nkgvyvqcmybm95ncwxb3zvrzbg2gz8kx6g6350v";
   };
 
-  buildInputs = [ go which ];
+  buildInputs = [ go which removeReferencesTo makeWrapper ];
```

Comment:
@Mic92 I tried replacing `stdenv.mkDerivation` with `buildGoPackage` and adding `goPackagePath = "golang.org/x/tools";` but I get the following error when trying to build:
```
...
./hack/verify-open-ports.sh: interpreter directive changed from "/bin/bash" to "/nix/store/53h800j8kgpj0a349f7wxa5hgkj1vby2-bash-4.4-p12/bin/bash"
./hack/verify-upstream-commits.sh: interpreter directive changed from "/bin/bash" to "/nix/store/53h800j8kgpj0a349f7wxa5hgkj1vby2-bash-4.4-p12/bin/bash"
configuring
grep: Invalid range end
building
make: *** No rule to make target 'build'.  Stop.
builder for ‘/nix/store/pf9pqx8a8hh1b2l6nsmsv3apwk1478z5-openshift-origin-1.5.0.drv’ failed with exit code 2
```